examples/05_stable_diffusion/README.md

```diff
@@ -56,7 +56,7 @@ By default, `compile_alt.py` does not include model weights (constants) with the
 
 #### Alternative pipeline
```

Comment:
Maybe move both "alternative build script" and "alternative pipeline" at the end of this README?

Reply:
IMHO it's better to have separate PRs for fixes and refactoring. This PR focuses on fixing script name `demo.py` -> `demo_alt.py`